### PR TITLE
Clarify behavior impacted by protocolMode

### PR DESCRIPTION
**Pull request by jasonnutter**

### CODE REVIEW

**Review by tnorling — APPROVED** [review]

File: lib/msal-browser/docs/initialization.md [hunk 1]
```diff
@@ -62,6 +62,12 @@ const msalConfig = {
 };
 ```
 
+**Note**: The `protocolMode` configuration option changes the following behavior:
```
Comment (sameerag) [inline lib/msal-browser/docs/initialization.md]: Nit: Can we add more context on why we added this? Any spec we can link to?

**Review by sameerag — APPROVED** [review]
Approved, I think more detail will help though.

File: lib/msal-browser/docs/initialization.md [hunk 2]
```diff
@@ -62,6 +62,12 @@ const msalConfig = {
...
+
+- Authority metadata (since `v2.4.0`):
+  - When set to `OIDC`, the library will not include `/v2.0/` in the authority path when fetching authority metadata.
+  - When set to `AAD`, the library will include `/v2.0/` in the authority path when fetching authority metadata.
```
Comment (jasonnutter) [inline lib/msal-browser/docs/initialization.md]:
```suggestion
  - When set to `AAD` (the default value), the library will include `/v2.0/` in the authority path when fetching authority metadata.
```

File: lib/msal-browser/docs/initialization.md (hunk 1 above)
Comment (jasonnutter) [reply lib/msal-browser/docs/initialization.md]: No specs, no. Can add more context as to the purpose.

Comment (jasonnutter) [inline lib/msal-browser/docs/initialization.md]:
```suggestion
**Note**: The `protocolMode` configuration option, which tells MSAL whether to enable AAD-specific quirks, changes the following behavior:
```

**Pull request merged**